cpp/src/arrow/adapters/arrow-rados-cls/README.md

```diff
@@ -76,3 +76,31 @@ docker-compose run --service-ports ubuntu-cls-demo
 # Development
 
 Check out these [instructions](docs/contributing.md) for setting up a local development environment.
+
+## Set up the development environment using docker
```

Comment:
Can you add this guide in the contributing.md instead of here ?

Reply:
Done